storyscript/hub/sdk/ServiceWrapper.py

```diff
@@ -0,0 +1,132 @@
+import json
+from uuid import UUID
+
+from storyscript.hub.sdk.GraphQL import GraphQL
+
+from storyscript.hub.sdk.service.ServiceData import ServiceData
+
+
+class UUIDEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, UUID):
+            return obj.hex
+        return json.JSONEncoder.default(self, obj)
+
+
+class ServiceWrapper:
+    """
+    The ServiceWrapper provides an improved way to access storyscript hub services
+    """
+    def __init__(self, services=None):
+        self.services = {}
+        self.reload_services(services)
+
+    @classmethod
+    def from_dict(cls, dictionary=None):
+        services = []
+
+        if dictionary is not None:
+            services = dictionary
+
+        return cls(services)
+
+    @classmethod
+    def from_json(cls, jsonstr=None):
```

Comment:
```suggestion
    def from_json(cls, jsonstr):
```

`jsonstr` shouldn't be optional.